proposal.rst

```diff
@@ -152,43 +154,51 @@ Motivations
 
 **Why have you chosen this particular project?**
 
-FILLME
+I want to participate in this project as it would be a learning experience for me and I think that it will be a good opportunity to identify and understand my skill set; improve it, and use it to solve real life problems.
+
+Plus I will be able to gain knowledge of different fields and learn under the guidance of experts.
+It would also help me fulfill my goal to use my->'Love of coding' to really make a difference by helping others.  
+
 
 **Who will benefit from your project and in what ways?**
 
 .. For example, think about the likely user-base, what they currently have to
 .. do and how your project will improve things for them.
 
-FILLME
+My project would benefit the majority of people that spend a lot of time being online. It would also help students
+
+Project Details:
+
+Idea Title: Projects related to testing
+Idea Link: https://trac.xapian.org/wiki/GSoCProjectIdeas#ProjectsRelatedtoTesting
```

Comment:
A proposal needs to be much more than just some details about yourself and a link to a project idea (or worse a group of project ideas).

You need to decide which project idea from that group you want to do, then dig into what implementing the project would involve, start to work out how you'd implement it, and show us that you understand what you're proposing to do.